src/Report/AppendQuery/Returns.php

```diff
@@ -95,6 +95,17 @@ public function getQueryBuilder()
 			}
 		}
 
+		// Filter brands
+		if ($this->_filters->exists('brand')) {
```

Comment:
Since the filter uses the `ModifyQueryInterface`, can we not loop through the filters and do that check like with did in the event listener in commerce? We can probably do that in this class rather than an event listener here though